content/library_handles.tex

```diff
@@ -25,8 +25,8 @@
 Handle of type \CTYPE{shmem\_team\_t} that corresponds to a team of \acp{PE}
 that share a memory domain. When this handle is used by some \ac{PE},
 it will refer to the team of all \acp{PE} that would return a non-null
-pointer from \FUNC{shmem\_ptr} for symmetric objects on that \ac{PE},
-and vice versa. This means that symmetric objects on each \ac{PE} are
+pointer from \FUNC{shmem\_ptr} for all symmetric heap objects on that \ac{PE},
+and vice versa. This means that symmetric heap objects on each \ac{PE} are
```

Comment:
Suggestion to replace "and vice versa" with a sentence or two explaining that for a PE to be in a particular SHMEM_TEAM_SHARED that the shmem_ptr relationship must go in both directions (local <-> remote PE). It may help to also update the above sentence to reference local and remote PEs, for example "would return a non-null pointer from a call to shmem_ptr by the local PE for all symmetric heap objects at a given target PE."

Reply:
How about this text:

> `SHMEM_TEAM_SHARED` refers to the team of all PEs that would mutually
> return a non-null address from a call to `shmem_ptr` for all symmetric heap
> objects.  That is, for every PE in `SHMEM_TEAM_SHARED`, `shmem_ptr`
> must return a non-null pointer to the local PE for all symmetric heap objects
> on all target PEs in the team.

"target PEs" would include the "local" PE in this case... so maybe the word, "target" isn't necessary...